lib/ansible/cli/__init__.py

```diff
@@ -798,3 +798,20 @@ def _play_prereqs(options):
         variable_manager.options_vars = load_options_vars(options, CLI.version_info(gitinfo=False))
 
         return loader, inventory, variable_manager
+
+    def _load_callback(self):
+
+        from ansible.plugins.loader import callback_loader
+        if self.callback is None:
+            self.callback = C.DEFAULT_STDOUT_CALLBACK
+        if self.callback not in callback_loader:
+            raise AnsibleError("Invalid callback for stdout specified: %s" % self.callback)
+        else:
+            self.callback = callback_loader.get(self.callback)
```

Comment:
Should this be restricted to callbacks with `CALLBACK_TYPE = 'stdout'`?

Reply:
yes, was assuming caller was correct, but we probably should check here, tqm does check but that might be too late.